imsc1-styling.xsd

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="utf-8"?>
+<xs:schema targetNamespace="http://www.w3.org/ns/ttml/profile/imsc1#styling" xml:lang="en"
+    xmlns:xs="http://www.w3.org/2001/XMLSchema"
+    xmlns:itts="http://www.w3.org/ns/ttml/profile/imsc1#styling">
+    
+    <xs:attribute name="fillLineGap">
+        <xs:annotation>
+            <xs:documentation>Controls the application of background between successive line areas.</xs:documentation>
+        </xs:annotation>
+        <xs:simpleType>
+            <xs:restriction base="xs:token">
```

Comment:
I'd be interested to know why you didn't use `base="xs:boolean"` here?

Reply:
@nigelmegitt Good question. The resaon is that xs:boolean also allows `1` and `0` as legal literals (to represent `true` or `false`). But `1` and `0` are not allowed as values for `itts:fillLineGap`. Therefore the type `xs:boolean` is not tight enough.

Reply:
Thanks that's good to know - I had `xs:boolean` in the LIT and when I fixed it to use these tokens I got a test error, because the value `True` was no longer accepted. Changing it to `'true'` worked...